app/models/concerns/roleable.rb

```diff
@@ -48,4 +50,15 @@ def set_permission(permission_name, value)
     save_and_refresh_staff_groups!
   end
 
+  private
+
+  def auto_approve_user
+    reviewable = ReviewableUser.where(status: Reviewable.statuses[:pending]).find_by(target: self)
```

Comment:
```suggestion
    reviewable = ReviewableUser.find_by(target: self, status: Reviewable.statuses[:pending])
```
This is equivalent I think and slightly shorter.

Reply:
You could also do `if reviewable = ...` and cut out a line.